packages/ng/form-field/form-field.component.html

```diff
@@ -1,20 +1,42 @@
-<label
-	class="formLabel"
-	[class.is-error]="invalid"
-	id="{{id}}-label"
-	for="{{id}}"
-	[class.u-mask]="hiddenLabel"
-	attr.aria-hidden="{{hiddenLabel}}"
->
-	<ng-container *luPortal="label"></ng-container
-	><!--
+<fieldset class="form-fieldset" *ngIf="layout === 'fieldset'; else defaultTpl" [class.mod-S]="size === 'S'">
```

Comment:
Is `mod-S` the only use case here or can we make it generic (mod-X)?

Reply:
Indeed, `<fieldset>` in only used associated with radios in this context for now.